src/Nethermind/Nethermind.Runner/RunnerApp.cs

```diff
@@ -36,14 +36,13 @@ public class RunnerApp : RunnerAppBase, IRunnerApp
         protected override (CommandLineApplication, Func<IConfigProvider>, Func<string>) BuildCommandLineApp()
         {
             string pluginsDirectory = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "plugins");
-            Console.WriteLine($"Loading plugins from: {pluginsDirectory}");
             if (Directory.Exists(pluginsDirectory))
```

Comment:
the whole idea was to display information on the plugin directory in case it was missing - need to add if...else... and display the plugin dir if it is configured (non empty) but cannot be found